src/styles/_components/_forms.scss

```diff
@@ -113,7 +113,7 @@
         color: color('text', 'secondary');
         text-align: left;
 
-        border-bottom: 1px solid color('background', 'progress-bar');
```

Comment:
We should avoid doing this at all costs. A seemingly `progress-bar` color was being used in the signup form...